src/main/webapp/script.js

```diff
@@ -13,7 +13,58 @@
 // limitations under the License.
 
 function addThreads() {  // eslint-disable-line
-  fetch('/data').then((response) => response.text()).then((quote) => {
-    document.getElementById('thread-container').innerText = quote;
+  const url = '/data';
+  fetch(url).then((response) => response.json()).then((threadInfoList) => {
+    const threadList = document.getElementById('thread-container');
+    threadList.innerHTML = '';
+
+    let i;
+    for (i = 0; i < threadInfoList.length; i += 4) {
+      // create description
+      const ulid = 'ul' + i;
+      const lidescription = document.createElement('LI');
+      lidescription.innerText = 'Description';
+      lidescription.className = 'descriptionli';
```

Comment:
nit: I think class names usually use hyphens, so change to "description-li" here and in css.